cli/crates/cli/tests/compilation_error.rs

```diff
@@ -52,20 +81,31 @@ fn compilation_error() {
         "#,
     );
 
-    env.write_schema(COMPILATION_ERROR_RESOLVER_SCHEMA);
+    std::thread::sleep(std::time::Duration::from_secs_f64(0.5));
 
-    client.poll_endpoint_for_changes(30, 300);
+    // For now without watching before we investigate the issue.
+    env.grafbase_dev();
+
+    let client = env.create_client_with_longer_timeout().with_api_key();
+    client.poll_endpoint(30, 300);
 
-    let response = client.gql::<Value>(COMPILATION_ERROR_RESOLVER_MUTATION).send();
+    let response = client.gql::<Value>(COMPILATION_ERROR_RESOLVER_QUERY).send();
 
-    let errors: Option<Vec<String>> = dot_get_opt!(response, "errors");
+    let errors: Option<Vec<Value>> = dot_get_opt!(response, "errors");
 
     assert_eq!(errors.map(|errors| !errors.is_empty()), Some(true));
 
```

Comment:
See above, sleeping for 2 secs in a test is a rather large increase to the overall runtime